src/main/java/com/overops/plugins/service/impl/RenderService.java

```diff
@@ -80,13 +80,13 @@ public void render() {
         if (getCheckTotalErrors() || getCheckUniqueErrors()) {
```

Comment:
I don't understand the purpose of this `if` statement. I think this can be removed and this logic can be simplified.